ext/opentelemetry-ext-opentracing-shim/tests/testbed/test_active_span_replacement/test_threads.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+
+from concurrent.futures import ThreadPoolExecutor
+
+from ..otel_ot_shim_tracer import MockTracer
+from ..testcase import OpenTelemetryTestCase
+
+
+class TestThreads(OpenTelemetryTestCase):
+    def setUp(self):
+        self.tracer = MockTracer()
+        self.executor = ThreadPoolExecutor(max_workers=3)
```

Comment:
This is mostly symbolic (as most of the tests are supposed to be, a hyper light reproduction of a use case, rather than an *actual* test case). I'd be up for adding a comment here to prevent any confusion on the specific usage of `3` ;)

Reply:
I'll add a comment.